tests/testthat/test-fbref.R

```diff
@@ -406,16 +406,16 @@ test_that("fb_league_stats() for players works", {
   testthat::skip_on_cran()
   testthat::skip_on_ci()
   expected_player_shooting_cols <- c("Rk", "Player", "Player_Href", "Nation", "Pos", "Squad", "Age", "Born", "Mins_Per_90", "Gls_Standard", "Sh_Standard", "SoT_Standard", "SoT_percent_Standard", "Sh_per_90_Standard", "SoT_per_90_Standard", "G_per_Sh_Standard", "G_per_SoT_Standard", "Dist_Standard", "FK_Standard", "PK_Standard", "PKatt_Standard", "xG_Expected", "npxG_Expected", "npxG_per_Sh_Expected", "G_minus_xG_Expected", "np:G_minus_xG_Expected", "Matches", "url")
-  epl_player_shooting_22 <- fb_league_stats(
+  single_player_shooting_22 <- fb_league_stats(
```

Comment:
realized this variable has `epl_` in it but we're scraping for Brazil. i've renamed it to `single_` to implicitly reflect its usage for testing just 1 league